tex/plan.tex

```diff
@@ -0,0 +1,33 @@
+\chapter{Project Plan}
+
+\section{Meetings}
+Our goals were outlined by weekly meetings. We regularly met with Jacob Graff, our advisor throughout the development of Extend.
+Jacob served as a sounding board whenever Extend's fundamental design philosophy was debated, and as a guide as we determined whether we were on track. We used any leftover time on those days to set goals for the upcoming week and pair program if time permitted.
+\newline \newline
+Our team also met weekly on Fridays to further discuss the progression of Extend. In the first half of the semester, the discussions were primarily philosophical, as decisions had to be made about the language grammar and behavior of certain Extend artifacts prior to development. In the second half, time was devoted to ironing out the development timeline, discussing bugs, and making compiler implementation decisions.
+
+\section{Development Workflow}
+
+  \subsection{Github \& Travis CI}
+  Our development and documentation were all done entirely through version control to maximize independent productivity.
+  New features were introduced to the master branch through pull requests, and the team collectively had to approve the code in order to merge each feature permanently.
```

Comment:
Factually the team did not have to collectively approve. It is more along the lines of review to ensure code quality.